### PR TITLE
docs(release): add v1.3.3 release notes

### PR DESCRIPTION
**Pull request by jpadilla**
## Description

Release for upcoming v1.3.3

## References

https://github.com/openfga/openfga/compare/v1.3.2...HEAD

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.3.3] - 2023-10-04
+
```
Comment (adriantam) [inline CHANGELOG.md:12]: Curious do we need something like 
```
[Full changelog](https://github.com/openfga/openfga/compare/v1.3.2...v1.3.3)
```
And don't know why we have it for some but not others

File: CHANGELOG.md [hunk 2]
```diff
@@ -620,7 +634,7 @@ no tuple key instead.
 ### Changed
 * Env variables have a new mappings.
 
-  Please refer to the [`.config-schema.json`](https://github.com/openfga/openfga/blob/main/.config-schema.json) file for a description of the new configurations or `openfga run -h` for the CLI flags. Env variables are   mapped by prefixing `OPENFGA` and converting dot notation into underscores (e.g. `datastore.uri` becomes `OPENFGA_DATASTORE_URI`). 
+  Please refer to the [`.config-schema.json`](https://github.com/openfga/openfga/blob/main/.config-schema.json) file for a description of the new configurations or `openfga run -h` for the CLI flags. Env variables are   mapped by prefixing `OPENFGA` and converting dot notation into underscores (e.g. `datastore.uri` becomes `OPENFGA_DATASTORE_URI`).
```
Comment (adriantam) [inline CHANGELOG.md:637]:
```suggestion
  Please refer to the [`.config-schema.json`](https://github.com/openfga/openfga/blob/main/.config-schema.json) file for a description of the new configurations or `openfga run -h` for the CLI flags. Env variables are mapped by prefixing `OPENFGA` and converting dot notation into underscores (e.g. `datastore.uri` becomes `OPENFGA_DATASTORE_URI`).
```

**Review by adriantam — COMMENTED** [review]
Can we confirm whether we need to update any of the helm chart before we release?

File: CHANGELOG.md (hunk 1 above)
Comment (jpadilla) [reply CHANGELOG.md:12]: ah yeah i can do that.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,31 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+
+* Reduce use of GOB in encoded cache key ([#1029](https://github.com/openfga/openfga/pull/1029))
+
+### Changed
+
+* Move standalone server config defaults ([#1036](https://github.com/openfga/openfga/pull/1036))
+
+* Persist Authorization Models serialized protobuf in the database ([#1028](https://github.com/openfga/openfga/pull/1028))
+
+  In the next series of releases will be progressively introducing changes via code and database migrations that will allow authorization models to be stored in a single database row.
```
Comment (adriantam) [inline CHANGELOG.md:31]:
```suggestion
  In the next series of releases will progressively introduces changes via code and database migrations that will allow authorization models to be stored in a single database row.
```

**Comment by jpadilla** [comment]
> Can we confirm whether we need to update any of the helm chart before we release?

@adriantam nothing that needs to be updated before, but I will update the version there after we release.

**Review by adriantam — APPROVED** [review]

**Pull request merged**